ads/uzou.js

```diff
@@ -33,6 +32,7 @@ export function uzou(global, data) {
   };
 
   const widgetParams = parseJson(data['widgetParams']);
+  const akamaiHost = widgetParams['akamaiHost'] || 'speee-ad.akamaized.net';
```

Comment:
We usually ask the contributor to wrap url params with `#encodeURIComponent()`, but since it is the host, I believe you won't want that. @send to confirm

Reply:
And simply to make sure the new change is documented your side.